app/src/main/res/values/styles.xml

```diff
@@ -35,4 +38,31 @@
 
     <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.MaterialComponents.Light"/>
 
+    <style name="CardShapeAppearance">
```

Comment:
```suggestion
    <style name="ShapeAppearance.Sunflower.Card">
```

Reply:
And move shape appearances to `shape.xml`

Reply:
Updated style name and added parent of ShapeAppearance.MaterialComponents

Reply:
Resolved in #477